windows/src/support/texteditor/Editor.vcxproj

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We need to make sure that output folders are per the standard layout (#5405), e.g. see IMSample at https://github.com/keymanapp/keyman/pull/5405/files#diff-7b13b71a59f91cf27e75c6e1220623dcadfa6aa40564f72fe358cfc0a4370d32